pkg/sshutil/sshutil_windows_amd64.go

```diff
@@ -0,0 +1,9 @@
+package sshutil
+
+import (
+	"github.com/intel-go/cpuid"
+)
+
+func detectAESAcceleration() bool {
+	return cpuid.HasFeature(cpuid.AES)
```

Comment:
I guess we can use this on macOS and Linux too?

Reply:
Possibly, not sure if it needs to check for OS "support" too ?

It might also be available in go: https://pkg.go.dev/internal/cpu

Reply:
Also I don't know how things work on Windows arm, so left it.